.claude/skills/feature-dev/references/snapshot.md

```
@@ -0,0 +1,84 @@
+| LocalAuth | local_auths | account, encodedPassword, memberKey |
+| OAuth | oauths | provider(KAKAO/NAVER), providerId, memberKey |
+| LoginHistory | login_histories | memberKey, ip, loginAt |
+| CareRelationship | care_relationships | caregiverKey, wardKey, role(GUARDIAN/MANAGER) |
+| CareInvitation | care_invitations | inviterKey, receiverKey, wardKey, status(PENDING/ACCEPTED/REJECTED/EXPIRED), createdAt |
+| GpsHistory | gps_histories | wardMemberKey, latitude, longitude, recordedAt |
+| WardDeviceToken | ward_device_tokens | wardKey(UUID, UNIQUE), token(UUID, UNIQUE), createdAt, expiresAt |
+| MembersTermsAgreement | members_terms_agreements | memberKey, agreedAt |
+
+## Redis 키 구조
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)